.bazelrc

```diff
@@ -23,3 +23,5 @@ build --cxxopt="-DNDEBUG"
 
 # Options from ./configure
 try-import %workspace%/.reverb.bazelrc
+
+build --features=-supports_dynamic_linker
```

Comment:
This was necessary due to https://github.com/bazelbuild/bazel/issues/4341.
I also had to use `bazel==4.1.0rc04` -- I tried 4.0.0 (the latest homebrew version)  and that didn't work for me for some reason.